features/rule-mgt/org.wso2.carbon.identity.rule.management.server.feature/resources/identity/rulemeta/flows.json

```diff
@@ -39,6 +39,18 @@
             {
               "name": "userInitiatedPasswordUpdate",
               "displayName": "User initiated password update"
+            },
+            {
+              "name": "adminInitiatedRegistrationWithPassword",
+              "displayName": "Admin initiated registration with password"
+            },
+            {
+              "name": "applicationInitiatedRegistrationWithPassword",
+              "displayName": "Application initiated registration with password"
+            },
+            {
+              "name": "userInitiatedRegistrationWithPassword",
+              "displayName": "User initiated registration with password"
             }
```

Comment:
Need to change these again. 

  {
    "name": "adminInitiatedRegistration",
    "displayName": "Admin initiated registration"
  },
  {
    "name": "applicationInitiatedRegistration",
    "displayName": "Application initiated registration"
  },
  {
    "name": "userInitiatedRegistration",
    "displayName": "User initiated registration"
  }